config.json

```diff
@@ -38,7 +38,7 @@
       }
     },
     "pnpm": {
-      "default": "8.13.1+sha1.90f9b2bb3ed58632bcb7b13c3902d6873ee9501c",
+      "default": "8.14.1+sha1.d039b38e0b20ad012ed548e44267b8d4c88b447a",
```

Comment:
Is this config still used?

I thought that corepack doesn't pin the global version of package managers anymore and instead fetches the latest?

Reply:
It's used when the environment disable the auto lookup: https://github.com/nodejs/corepack/blob/9bee4150815113d97f0bd77d62c8d999cfd68ad3/README.md#L222-L224